sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -0,0 +1,378 @@
+string endpoint = "<endpoint>";
+string subscriptionKey = "<subscriptionKey>";
+var client = new TextAnalyticsClient(new Uri(endpoint), subscriptionKey);
+```
+
+#### Get Subscription Key
+
+Use the [Azure CLI][azure_cli] snippet below to get the subscription key from the Text Analytics resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+Alternatively, you can get the endpoint and subscription key from the resource information in the [Azure Portal][azure_portal].
+
+#### Create TextAnalyticsClient with Azure Active Directory Credential
+
+Client subscription key authentication is used in most of the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below,
+or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
+
+```PowerShell
+Install-Package Azure.Identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and [grant access][aad_grant_access] to Text Analytics by assigning the `"Cognitive Services User"` role to your service principal.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+```C# Snippet:CreateTextAnalyticsClientTokenCredential
+string endpoint = "<endpoint>";
+var client = new TextAnalyticsClient(new Uri(endpoint), new DefaultAzureCredential());
+```
+
+## Key concepts
+
+### TextAnalyticsClient
+A `TextAnalyticsClient` is the primary interface for developers using the Text Analytics client library.  It provides both synchronous and asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction. 
+
+### Text Input
+A **text input**, sometimes called a **document**, is a single unit of input to be analyzed by the predictive models in the Text Analytics service.  Operations on `TextAnalyticsClient` may take a single text input or a collection of inputs to be analyzed as a batch.
+
+### Operation Result
+An operation result, such as `AnalyzeSentimentResult`, is the result of a Text Analytics operation, containing a prediction or predictions about a single text input.  An operation's result type also may optionally include information about the input document and how it was processed.
+
+### Operation Result Collection
+ An operation result collection, such as `AnalyzeSentimentResultCollection`, is a collection of operation results, where each corresponds to one of the text inputs provided in the input batch.  A text input and its result will have the same index the input and result collections.  An operation result collection may optionally include information about the input batch and how it was processed.
```

Comment:
"A text input and its result will have the same index as the input and result collections" ?

Reply:
I missed an "in"!